### PR TITLE
Fix NLTE Normalization - issue #784

### PR DESCRIPTION
**Pull request by livnehra**
Fix NLTE normalization according to @ssim suggestion in the discussion in #784

### CODE REVIEW

File: tardis/plasma/properties/partition_function.py [hunk 1]
```diff
@@ -190,7 +190,7 @@ def _main_nlte_calculation(
                     else:
                         raise e
                 general_level_boltzmann_factor[i].ix[species] = \
-                    level_boltzmann_factor
+                    level_boltzmann_factor * g[species][0] / level_boltzmann_factor[0]
```
Comment (wkerzendorf) [inline tardis/plasma/properties/partition_function.py]: @livnehra can you make this `g.loc[species, 0]`

**Comment by wkerzendorf** [comment]
@livnehra I don't think that should run - but maybe I'm wrong

File: tardis/plasma/properties/partition_function.py [hunk 2]
```diff
@@ -190,7 +190,7 @@ def _main_nlte_calculation(
                     else:
                         raise e
                 general_level_boltzmann_factor[i].ix[species] = \
-                    level_boltzmann_factor * g[species][0] / level_boltzmann_factor[0]
+                    level_boltzmann_factor * g.loc[species][0] / level_boltzmann_factor[0]
```
Comment (wkerzendorf) [inline tardis/plasma/properties/partition_function.py]: `.loc[species, 0]`

Comment (livnehra) [reply tardis/plasma/properties/partition_function.py]: That doesn't work...

Comment (wkerzendorf) [reply tardis/plasma/properties/partition_function.py]: hmm  - that's interesting. I'll check it tomorrow.

Comment (wkerzendorf) [reply tardis/plasma/properties/partition_function.py]: yes you're right - I screwed things up in my mind. I think the way you do it now is good.

Comment (livnehra) [reply tardis/plasma/properties/partition_function.py]: Remember, for example species=(14,1) for Si II

**Comment by wkerzendorf** [comment]
@livnehra the tests fail - your change somehow breaks the code - have a look at the continuous integration to see what's wrong.

**Comment by livnehra** [comment]
Hopefully should pass the tests now.

**Comment by livnehra** [comment]
So the fails now are due to NLTE results changing, which makes sense if the previous results were obtained with the erroneous g_ratio_matrices.

**Comment by wkerzendorf** [comment]
@livnehra - okay I'm ready to update the reference data. I still think it would be good to get a spectrum beforehand.

**Comment by unoebauer** [comment]
@wkerzendorf, @livnehra - what's the idea about proceeding with this PR?

File: tardis/io/atomic.py [hunk 3]
```diff
@@ -442,7 +442,7 @@ def _create_collision_coefficient_matrix(self):
                 C_ul_matrix[level_number_lower, level_number_upper, :] = line.values[2:]
                 delta_E_matrix[level_number_lower, level_number_upper] = line['delta_e']
                 #TODO TARDISATOMIC fix change the g_ratio to be the otherway round - I flip them now here.
```
Comment (chvogl) [inline tardis/io/atomic.py]: It would be possible to remove the TODO now that it has been addressed.

**Pull request merged**